pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -286,13 +314,41 @@ var _ = g.Describe("[Feature:Machines] Autoscaler should", func() {
 				nodeCounter++
 			}
 
-			glog.Infof("Expecting at least one new node to come up. Initial number of node group nodes: %d. Current number of nodes in the group: %d", nodeGroupInitialTotalNodes, nodeCounter)
-			return nodeCounter > nodeGroupInitialTotalNodes, nil
+			glog.Infof("Expecting %v nodes. Initial number of node group nodes: %d. Current number of nodes in the group: %d", initialNumberOfReplicas0+2, nodeGroupInitialTotalNodes1, nodeCounter)
```

Comment:
should this be "Expecting %d nodes"?

Reply:
I can use %d but what is harm in using %v.
https://stackoverflow.com/questions/41159169/why-not-use-v-to-print-int-and-string

Reply:
None really. And if you change the type, the diff can be smaller.